projects/aws/upgrader/upgrade/containerd_upgrade.go

```diff
@@ -0,0 +1,48 @@
+package upgrade
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/eks-anywhere-build-tooling/tools/version-tracker/pkg/util/logger"
+)
+
+func (u *Upgrader) ContainerdUpgrade(ctx context.Context) error {
```

Comment:
Should we add comments to this business logic to explain what is happening as part of this flow?